WordSearchTest.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class WordSearchTest {

```

Comment:
What about tests for other cases, like empty grid or grid that contains no words from the dictionary?